core/trino-main/src/main/java/io/trino/server/GenerateTraceTokenRequestFilter.java

```diff
@@ -1,63 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.server;
-
-import com.google.common.annotations.VisibleForTesting;
-import io.airlift.http.client.HttpRequestFilter;
-import io.airlift.http.client.Request;
-
-import java.util.concurrent.atomic.AtomicLong;
-
-import static io.airlift.http.client.Request.Builder.fromRequest;
-import static io.airlift.http.client.TraceTokenRequestFilter.TRACETOKEN_HEADER;
-import static java.util.Locale.ENGLISH;
-import static java.util.Objects.requireNonNull;
-import static java.util.UUID.randomUUID;
-
-public class GenerateTraceTokenRequestFilter
```

Comment:
It looks like most of the changes in this commit should be in the "Remove Airlift's trace token" commit